src/SuperDumpModels/SDFileAndLineNumber.cs

```diff
@@ -9,7 +9,23 @@ public class SDFileAndLineNumber : IEquatable<SDFileAndLineNumber>, ISerializabl
 		public int Line;
 
 		public string FileName() {
-			return Path.GetFileNameWithoutExtension(File);
+			return File;
+		}
+
+		public bool IsLinkAvailable() {
+			return File.Contains("/agent/native") && File.Contains("sprint_");
```

Comment:
this is dynatrace specific. it's ok, but maybe we should mark it as such with a comment.